### DISCUSSION TITLE
Pronounciation of 'Vike'

### DISCUSSION THREAD

**Discussion by spdiswal** [vikejs/vike#1230] [Discussion] [answered]
What is the correct way to say _Vike_? I haven't found any official mentioning of this in text or videos.

Does it lean towards the pronounciation of _Vite_, i.e. 'veek'?
Or does it rhyme with bike/like, i.e. 'vaik'? Or perhaps even 'vai-key' like Nike? 😄

**Comment by brillout** [comment, marked as answer]
Like Viking, i.e. 'vaik' &mdash; contribution welcome to add an audio recording to `https://vike.dev/` 🙂

**Reply by spdiswal** [reply]
Cool, thanks for clarifying!
